consensus/src/traits.rs

```diff
@@ -152,6 +152,12 @@ pub trait ConsensusApi<
         .await;
     }
 
+    /// Send a broadcast to the DA comitee, stub for now
+    async fn send_da_broadcast<DAPROPOSAL: ProposalType<NodeType = TYPES>>(
```

Comment:
I think this might need a view number to determine the DA committee for this view.

Reply:
I will add view number to the function signature as a placeholder for now.